src/main/java/org/sagebionetworks/bridge/udd/helper/SnsHelper.java

```diff
@@ -0,0 +1,76 @@
+package org.sagebionetworks.bridge.udd.helper;
+
+import java.util.Map;
+
+import org.sagebionetworks.bridge.udd.accounts.AccountInfo;
+import org.sagebionetworks.bridge.udd.dynamodb.StudyInfo;
+import org.sagebionetworks.bridge.udd.s3.PresignedUrlInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.amazonaws.services.sns.AmazonSNSClient;
+import com.amazonaws.services.sns.model.MessageAttributeValue;
+import com.amazonaws.services.sns.model.PublishRequest;
+import com.amazonaws.services.sns.model.PublishResult;
+import com.google.common.collect.Maps;
+
+public class SnsHelper {
+    private static final Logger LOG = LoggerFactory.getLogger(SnsHelper.class);
+    
+    /**
+     * 11 character label as to who sent the SMS message. Only in some supported countries (not US):
+     * https://support.twilio.com/hc/en-us/articles/223133767-International-support-for-Alphanumeric-Sender-ID
+     */
+    public static final String SENDER_ID = "AWS.SNS.SMS.SenderID";
+    /** SMS type (Promotional or Transactional). */
+    public static final String SMS_TYPE = "AWS.SNS.SMS.SMSType";
+    /** SMS message type. */
+    public static final String SMS_TYPE_TRANSACTIONAL = "Transactional";
```

Comment:
Do we want to call SNS directly? Or do we want to call the Bridge API for sending SNS? The latter might be better for not having so much duplicate code.